kafka/topics/dev/attribute.properties

```diff
@@ -0,0 +1,6 @@
+name=event-store.dev_attribute_registry.events
+partitions=3
+replication.factor=3
```

Comment:
to save some 💰 this could be lowered to 1 in dev.
It can be done later, so not necessary for this PR